tests/test_enterprise/api/test_views.py

```diff
@@ -1229,6 +1229,7 @@ class TestEnterpriseCustomerViewSet(BaseTestEnterpriseAPIViews):
                 'enable_demo_data_for_analytics_and_lpr': False,
                 'enable_academies': False,
                 'enable_one_academy': False,
```

Comment:
Can we get one quick positive test here for active integrations?

Reply:
done!